docfx.json

```diff
@@ -176,7 +176,7 @@
       "author": {
         "docs/architecture/**/**.md": "nishanil",
         "docs/architecture/grpc-for-wcf-developers/**/**.md": "shirhatti",
-        "docs/core/**/**.md": "mairaw",
+        "docs/core/**/**.md": "tdykstra",
```

Comment:
Since we can't use the dotnet/docs GH team for author, I just rotated through each team member for /core, /framework, /standard/glob-loc, /standard/linq.

Reply:
I just checked the [contributor guide](https://docsmetadatatool.azurewebsites.net/allowlists#ms.author). We can use our team alias, `dotnetcontent` here instead.  That would be better than rotating assignments. Thoughts?

Reply:
Sure for ms.author, but even for author? It has to be a GitHub ID.

Reply:
Ahh. According to the docfx folks, `@dotnet/docs` should work.

Reply:
I did try that on line 179 in [this commit](https://github.com/dotnet/docs/pull/17389/commits/26c282fd90e304154164d972556868bd0c24747b#diff-f14cdab6f1d762de4c71972b81dd008cR179), but the build had a warning that it wasn't a valid GitHub account.

Reply:
*sigh* After I had the first response that GitHub teams were supported, one of the engineers wrote back to correct me that they aren't. Let's go with your original plan of cycling through each of us.